core/src/com/unciv/ui/trade/ExpanderTab.kt

```diff
@@ -1,45 +1,100 @@
 package com.unciv.ui.trade
 
+import com.badlogic.gdx.graphics.Color
+import com.badlogic.gdx.math.Interpolation
 import com.badlogic.gdx.scenes.scene2d.Touchable
-import com.badlogic.gdx.scenes.scene2d.ui.Skin
+import com.badlogic.gdx.scenes.scene2d.actions.FloatAction
 import com.badlogic.gdx.scenes.scene2d.ui.Table
+import com.badlogic.gdx.utils.Align
+import com.unciv.UncivGame
+import com.unciv.ui.utils.CameraStageBaseScreen
 import com.unciv.ui.utils.ImageGetter
 import com.unciv.ui.utils.onClick
 import com.unciv.ui.utils.toLabel
 
-class ExpanderTab(private val title:String,skin: Skin): Table(skin){
-    private val toggle = Table(skin) // the show/hide toggler
-    private val tab = Table() // what holds the information to be shown/hidden
-    val innerTable= Table() // the information itself
-    var isOpen=true
-
-    init{
-        toggle.defaults().pad(10f)
-        toggle.touchable= Touchable.enabled
-        toggle.background(ImageGetter.getBackground(ImageGetter.getBlue()))
-        toggle.add("+ $title".toLabel(fontSize = 24))
-        toggle.onClick {
-            if(isOpen) close()
-            else open()
+/**
+ * A widget with a header that when clicked shows/hides a sub-Table
+ * 
+ * Add content to be shown/hidden to [innerTable]!
+ * 
+ * @param title The header text, automatically translated
+ * @param initialOpen Should it start out opened? Defaults to `true`.
```

Comment:
I'm a believer in 'if a param needs an explanation, it either has hidden functionality or it's badly named'

Reply:
My guideline is 'do I find the tooltip useful' (and don't invest more than a few seconds for docs):
![image](https://user-images.githubusercontent.com/63000004/124369581-5cb76180-dc6d-11eb-9b26-49918b6ba9f8.png)

Methods often need less because Studio shows the signature anyway, so all parameter names and types are already there. Classes lack that part, so I try to mention them all - and in this case I failed, forgot the two I added later.